<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_ProjectNote</class>
 <widget class="QDialog" name="W_ProjectNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工程笔记</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>工程名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>创建时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="plainText">
         <string>作者: 
备注: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Retry</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>W_ProjectNote</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
